workflow/controller/operator_wfdefault_test.go

```diff
@@ -222,12 +222,11 @@ func TestWFDefaultWithWFTAndWf(t *testing.T) {
 	var resultSpec wfv1.WorkflowSpec
 	err := json.Unmarshal([]byte(storedSpecResult), &resultSpec)
 	assert.NoError(err)
-	cancel, controller := newControllerWithDefaults()
-	defer cancel()
-	controller.Config.WorkflowDefaults = wfDefault
-	_, err = controller.wfclientset.ArgoprojV1alpha1().WorkflowTemplates("default").Create(wft)
-	assert.NoError(err)
 	t.Run("SubmitSimpleWorkflowRef", func(t *testing.T) {
+		cancel, controller := newController(wft)
```

Comment:
The test was randomly failed with "workflow-template-submittable" not found.
Moved controller to local and pass WFT to the controller instead of making create api call